enhancements/machine-api/spot-instances.md

```diff
@@ -0,0 +1,359 @@
+---
+title: spot-instances
+authors:
+  - "@JoelSpeed"
+  - "@enxebre"
+reviewers:
+  - "@enxebre"
+  - "@bison"
+  - "@michaelgugino"
+approvers:
+  - "@enxebre"
+  - "@bison"
+  - "@michaelgugino"
+creation-date: 2020-02-04
+last-updated: 2020-02-04
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Spot Instances
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Enable OCP users to leverage cheaper, non-guaranteed instances to back Machine API Machines.
+
+## Motivation
+
+Allow users to cut costs of running OCP clusters on cloud providers by moving interruptible workloads onto non-guaranteed instances.
```

Comment:
:heart: I can imagine this saving our own CI nontrivial amounts of money, not to mention the obvious customer use cases.